docs: Translated readme into english, added module inventory (#12)
OKDP/OKDP: README.md

## ⚙️ Infrastructure Prerequisites

| Component     | Description                                                            | Helm Chart                                                         |
| ------------- | ---------------------------------------------------------------------- | ------------------------------------------------------------------ |
| cert-manager  | Kubernetes-native TLS certificate management (CNCF Graduated)          | [jetstack/cert-manager](https://charts.jetstack.io)                |
| ingress-nginx | NGINX Ingress Controller for HTTP/HTTPS routing                        | [ingress-nginx](https://kubernetes.github.io/ingress-nginx)        |
| CloudNativePG | PostgreSQL operator for automated cluster management (CNCF Sandbox)    | [cnpg/cloudnative-pg](https://cloudnative-pg.github.io/charts)     |
| Keycloak      | Identity and access management with OIDC support for all OKDP services | [codecentric/keycloakx](https://codecentric.github.io/helm-charts) |
| SeaweedFS     | S3-compatible distributed object storage for data lake and event logs  | [seaweedfs/helm](https://seaweedfs.github.io/seaweedfs/helm)       |

> > For testing purpose, you can deploy all infrastructure prerequisites at once with use the [okdp-prerequisites umbrella chart](https://github.com/OKDP/helm-handbook/tree/main/modules/prerequisites/okdp-prerequisites). **Do not use it for production cluster.**

Reviewer: I think the word "use" is not needed, I made a bad suggestion.
```suggestion
> For testing purpose, you can deploy all infrastructure prerequisites at once with the [okdp-prerequisites umbrella chart](https://github.com/OKDP/helm-handbook/tree/main/modules/prerequisites/okdp-prerequisites). **Do not use it for production cluster.**
```

Author: Got it. Thanks

<div align="center">

> <img src="https://raw.githubusercontent.com/OKDP/OKDP/main/logo/main/okdp-main.png" alt="OKDP Logo" width="320" />

Reviewer: Do not change the logo used, the "normal" logo is bad with Github light theme.

Author: Done. I kept the <img in order to give a size to the image because by default the image is too huge

## Infrastructure Prerequisites

| Component | Description | Helm Chart | Install Guide |
| --- | --- | --- | --- |
| cert-manager | Kubernetes-native TLS certificate management (CNCF Graduated) | [jetstack/cert-manager](https://charts.jetstack.io) | [Install Guide](https://github.com/OKDP/helm-handbook/tree/main/modules/prerequisites/cert-manager) |
| ingress-nginx | NGINX Ingress Controller for HTTP/HTTPS routing | [ingress-nginx](https://kubernetes.github.io/ingress-nginx) | [Install Guide](https://github.com/OKDP/helm-handbook/tree/main/modules/prerequisites/ingress-nginx) |
| CloudNativePG | PostgreSQL operator for automated cluster management (CNCF Sandbox) | [cnpg/cloudnative-pg](https://cloudnative-pg.github.io/charts) | [Install Guide](https://github.com/OKDP/helm-handbook/tree/main/modules/prerequisites/cloudnative-pg) |
| Keycloak | Identity and access management with OIDC support for all OKDP services | [codecentric/keycloakx](https://codecentric.github.io/helm-charts) | [Install Guide](https://github.com/OKDP/helm-handbook/tree/main/modules/prerequisites/keycloak) |

> | SeaweedFS | S3-compatible distributed object storage for data lake and event logs | [seaweedfs/helm](https://seaweedfs.github.io/seaweedfs/helm) | [Install Guide](https://github.com/OKDP/helm-handbook/tree/main/modules/prerequisites/seaweedfs) |

Reviewer: The prerequisites comes from each components and are not part of OKDP, we should remove it.

Author: This has already been removed in the latest changes

Reviewer: I still see them

Reviewer 2: I still see the infrastructure prerequisites too

Author: Oooh so your comment was referring to removing the entire prerequisites bloc? Huuummm.... for me this is an important info to have as `OKDP/OKDP` is suppose to be the entrypoint. 

As a general user, knowing directly the data components and the inventory Infrastructure prerequisites is not that bad to be... Do you guys really believe this bloc does not belong to the `OKDP/OKDP/README` ?

Reviewer: To rephrase what I said in my first comment, OKDP is modular, a user can install a single component, so why would they need all the prerequisites? The prerequisites to install depend on the data components the user installs. Therefore, in my opinion, the documentation for the prerequisites is found within the component's documentation.

Author: That is a strong argument there ! Listing them all upfront in the root README implies they're all required, wich is misleading.
You know what ! Lets keep as it is and if there is no objection, I will remote it. Thanks

Reviewer 2: For me, this creates confusion by suggesting that they all have to be installed, which contradicts the idea of ​​a modular platform, so it should be removed.

Author: Removed

> > To deploy all infrastructure prerequisites at once, use the [okdp-prerequisites umbrella chart](https://github.com/OKDP/helm-handbook/tree/main/modules/prerequisites/okdp-prerequisites).

Reviewer: The umbrella chart is an example and should not be used for production.

```suggestion
> For testing purpose, you can deploy all infrastructure prerequisites at once with use the [okdp-prerequisites umbrella chart](https://github.com/OKDP/helm-handbook/tree/main/modules/prerequisites/okdp-prerequisites). **Do not use it for production cluster.**
```

Author: So there is no way someone will use manual install guide for production purpose?

Reviewer: Like we did for TDP, the prerequisite we provide in OKDP repos are only example and we do not offer production ready prerequisites because our job is to make a Data platform and not a prerequisites platform.

Author: Makes sense. Thanks for clarifying !

| Component | Description | Repo |
| --- | --- | --- |
| OKDP Server | REST API backend for managing deployments, clusters, projects, GitOps repositories, and package catalogs | [okdp-server](https://github.com/OKDP/okdp-server) |
| OKDP UI | Web interface for deploying and monitoring all OKDP services, with GitOps and direct Kubernetes deployment modes | [okdp-ui](https://github.com/OKDP/okdp-ui) |

> The Control Plane is optional. All Data Platform components can be deployed independently via Helm without it.

---

## Getting Started

### Option 1: Sandbox (recommended)

The fastest way to explore OKDP. Spins up a full, pre-configured platform on [Kind](https://kind.sigs.k8s.io/) with a single command.

→ [okdp-sandbox](https://github.com/OKDP/okdp-sandbox)

### Option 2: Manual install

Deploy components individually on your own cluster using the Helm-based install guides.

1. Install infrastructure prerequisites: follow the [okdp-prerequisites guide](https://github.com/OKDP/helm-handbook/tree/main/modules/prerequisites/okdp-prerequisites) or install modules individually

> 2. Install the data platform components you need using their [install guides](#data-platform-components)

Reviewer: You can remove it, this is the scope for the README inside `helm-handbook`.

Author: Done. Kept only the reference to the repo

> To deploy the full stack at once, see the [helm-handbook](https://github.com/OKDP/helm-handbook) main README.

---

## Community & TOSIT

OKDP is supported by [TOSIT](https://tosit.fr) (The Open Source I Trust), initiated by DGFiP, Orange, and other organizations. The goal is a sovereign, powerful, and fully open-source data stack accessible to everyone.

- **Website**: [okdp.io](https://okdp.io)
- **Discussions**: [Mattermost OKDP (TOSIT)](https://framateam.org/tosit/channels/okdp)
- **Weekly technical meeting**: Every Wednesday at 10:00 CET
- **Docker images**: [Quay.io/okdp](https://quay.io/organization/okdp)
- **Java artifacts**: [Maven Central (io.okdp)](https://central.sonatype.com/namespace/io.okdp)
- **Contribute**: [okdp.io#community](https://okdp.io#community)

## Roadmap

See the [official roadmap](https://okdp.io/roadmap/) on okdp.io (v1.0.0 planned for June 2026).

## Contributing

See [CONTRIBUTING.md](CONTRIBUTING.md) for documentation standards and review process.

## Governance

See [GOVERNANCE.md](GOVERNANCE.md) for project governance, roles, and decision-making.

## License

> [Apache License 2.0](./LICENSE)

Reviewer: Like contributing and governance above

```suggestion
[Apache License 2.0](LICENSE)
```
